Tests/SonarRunner.Shim.Tests/PropertiesFileGeneratorTests.cs

```diff
@@ -246,6 +246,51 @@ public void FileGen_MissingFilesAreSkipped()
             logger.AssertSingleWarningExists(missingContentFile);
         }
 
+        [TestMethod]
+        [Description("Checks that the generated properties file contains additional properties")]
```

Comment:
The property file generator code hasn't changed, but there weren't any tests that checked that additional settings were written to the file.